src/NuGetGallery/App_Data/Files/Content/flags.json

```
@@ -52,6 +52,12 @@
       "SiteAdmins": false,
       "Accounts": [],
       "Domains": []
+    },
+    "NuGetGallery.LicenceMdRendering": {
+      "All": false,
```

Let's enable this feature for local development:

```suggestion
      "All": true,
```